ecosystem/plugin-search/src/client/utils/isFocusingTextControl.ts

```diff
@@ -0,0 +1,14 @@
+/**
+ * Determines whether the user is currently focusing a text control. In this case, the search plugin shouldn’t hijack any hotkeys because the user might be typing into a text field, using type-ahead search in a `select` element, etc.
+ */
+export function isFocusingTextControl(target: EventTarget): boolean {
+  if (target instanceof HTMLElement) {
+    return (
+      document.activeElement === target &&
+      (['TEXTAREA', 'SELECT', 'INPUT'].includes(target.tagName) ||
+        target.hasAttribute('contenteditable'))
+    )
+  } else {
```

Comment:
Else block for return is not needed

Reply:
I moved the return statement out of the else block.